### DISCUSSION TITLE
EssentialsX PVP Control

### DISCUSSION THREAD

**Discussion by sammikinsrox** [GriefPrevention/GriefPrevention#1930] [Q&A]
I am trying to use EssentialsX PVP control. Within the EssentialsX config, there is `disable.pvp: true/false` you can bypass the PVP disability and allow players to continue to PVP while it is disabled by using the `essentials.protect.pvp` permission.

I am trying to manage PVP based on user permission groups using the `essentials.protect.pvp` flag. But it seems GriefPrevention is overriding the settings setup with EssentialsX.

Is there a solution to defer control of PVP to EssentialsX, OR, is there a way to manage PVP access to specific permissions groups using GriefPrevention?

It seems GriefPrevention is a perfect solution for my server's needs but I'd like to be able to disable pvp from guests / default permissions level.

Thank you for your time!

**Comment by sammikinsrox** [comment]
Nevermind I see that this plugin has been deprecated to support GriefDefender.

**Reply by Jikoo** [reply]
GD replaces GP for Sponge, which was a port not affiliated with the Bukkit version. GP for Bukkit has not been discontinued and is still maintained.

**Reply by sammikinsrox** [reply]
Oh! That's amazing to hear!

**Comment by Jikoo** [comment]
GP ignores cancelled events and runs its PVP checks on lowest priority, so it should not ever enable PVP if it is already blocked. If you're having issues, please open a bug report including any relevant config/perm details.

Alternately, if you're happy with another system and don't want to use GP's, just turn off GP's PVP management in the config.

**Reply by sammikinsrox** [reply]
Okay, I'll try to submit a bug report for you. Thank you for your help!
